packages/story-editor/src/utils/test/nativeCopyPasteExpected.js

```diff
@@ -51,9 +51,10 @@ describe('nativeCopyPasteExpected', () => {
     expect(nativeCopyPasteExpected()).toBe(true);
   });
 
-  // Disable reason: jsdom does not actually support checking for contenteditable.
-  // See https://github.com/jsdom/jsdom/issues/1670
-  //eslint-disable-next-line jest/no-disabled-tests
+  /* eslint-disable-next-line jest/no-disabled-tests -- 
```

Comment:
You can fix the lint errors here with `npx eslint --fix packages/story-editor/src/utils/test/nativeCopyPasteExpected.js`

Or run `npm run lint:js:fix` locally to autofix _all files_ (takes a while)